waraich1/trenddit-frontend
# Sidebar, Routing and Logo

This introduces the general screen layout, a sidebar, and some routing along with our apps logo. I've tried to make it sort of dynamic so you can stretch it around and nothing should break.


**Test Plan:**
- Ensure Trenddit Logo and Text saying Trenddit is located in the top left corner of the screen on an orange banner
- Ensure that there are three bars on the top right of the screen to open up the sidebar
- Press the bars and ensure a sidebar opens with options for "Subreddit", Trends", "User" and "Logout".
- Once the sidebar is open, ensure that pressing the X button to closes it
- Click on Subreddit. The sidebar should close and nothing should change as I have temporarily set the home page to be the same as the subreddit page
- Reopen the menu bar and click on Trends. You should be redirected to a new address that ends with /trends in the URL.
- Reopen the menu bar and click on User. You should be redirected to a new address that ends with /user in the URL.
- Reopen the menu bar and click on Logout. The application should go to an error screen as I haven't implemented anything for Logout yet. For now, it will just serve as a placeholder


Aside from manually testing, If you all have suggestions when it comes to code clean-up or organization please do let me know. I just wanted to get this up, but I might go back and do some general clean-up if I have time
